stablehlo/dialect/StablehloOps.td

```diff
@@ -2097,13 +2097,22 @@ def StableHLO_UnaryEinsumOp: StableHLO_Op<"unary_einsum", [Pure]>, BASE_EinsumOp
 def StableHLO_FftOp: StableHLO_Op<"fft", [InferTensorType, Pure]> {
   let summary = "Fast fourier transform operator";
   let description = [{
-    Returns the fast-fourier-transform of the input array.
+    Performs the forward and inverse Fourier Transforms for real and complex
+    inputs/outputs.
 
-    See
-    https://www.tensorflow.org/xla/operation_semantics#fft.
+    See:
+    https://github.com/openxla/stablehlo/blob/main/docs/spec_draft.md#stablehlofft
+
+    Example:
+    ```mlir
+    %result = "stablehlo.fft"(%operand) {
+      fft_length = dense<4> : tensor<1xi64>,
+      fft_type = #stablehlo<fft_type FFT>
+    } : (tensor<4xcomplex<f32>>) -> tensor<4xcomplex<f32>>
+    ```
```

Comment:
Get most recent changes, this op has pretty print now and the following example at head:

```
    Example:

    ```mlir
    %0 = stablehlo.fft %arg0, type = RFFT, length = [9] : (tensor<3x9xf32>) -> tensor<3x5xcomplex<f32>>
    ```
```

This can be updated to use ODS names.

Reply:
Thanks for the heads up!